go/engine/saltpack_sender_identify.go

```diff
@@ -88,6 +89,7 @@ func (e *SaltPackSenderIdentify) identifySender(ctx *Context) (err error) {
 		ForceRemoteCheck:      e.arg.forceRemoteCheck,
 		NoErrorOnTrackFailure: true,
 		Reason:                e.arg.reason,
+		UserAssertion:         e.arg.userAssertion,
```

Comment:
this is OK for now, but long-term, we should try a strategy that supports OR-assertions. with a rebase, it shouldn't be too bad.